peer/mrunoncemap.go

```diff
@@ -107,12 +105,13 @@ func (m *mruNonceMap) Add(nonce uint64) {
 // This function is safe for concurrent access.
 func (m *mruNonceMap) Delete(nonce uint64) {
 	m.mtx.Lock()
-	defer m.mtx.Unlock()
 
```

Comment:
Same thing here regarding the lock around the critical section and spacing consistency.